src/utils_SymKLIEPLoss.jl

```diff
@@ -22,7 +22,9 @@ end
 CoordinateDescent.numCoordinates(f::CDSymKLIEPLoss) = f.p
 
 function CoordinateDescent.initialize!(f::CDSymKLIEPLoss, x::SparseIterate)
-    mul!(f.rx, transpose(f.Ψx),-x)
+    mul!(f.rx, transpose(f.Ψx), x)
+    f.rx *= -1.
```

Comment:
This should be 
`f.rx .*= -1.`

Notice the difference

```
julia> function f(a) 
         a *= -1.
         a
       end
f (generic function with 1 method)

julia> function g(a)
         a .*= -1.
         a
       end
g (generic function with 1 method)

julia> a = randn(1000000);

julia> @time f(a);
  0.005483 seconds (1.56 k allocations: 7.720 MiB)

julia> @time f(a);
  0.003113 seconds (6 allocations: 7.630 MiB)

julia> @time f(a);
  0.019711 seconds (6 allocations: 7.630 MiB, 83.54% gc time)

julia> @time g(a);
  0.017159 seconds (20.62 k allocations: 1.010 MiB)

julia> @time g(a);
  0.001316 seconds (4 allocations: 160 bytes)

julia> @time g(a);
  0.001246 seconds (4 allocations: 160 bytes)
```

The first time the function is compiled, but on the second run it is faster and allocates less memory.

Reply:
Changes made. Thanks! I'd never have imagined that a . could make such a difference.

Reply:
This is a useful link to read
https://docs.julialang.org/en/v1/manual/performance-tips/index.html